src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -203,6 +203,44 @@ controlPlane:
 
 Please note that this only applies to the API Server and that etcd, the scheduler, and the controller manager remain inaccessible in cloud environments.
 
+#### Control plane monitoring for Rancher RKE1 [#rancher]
+
+Clusters deployed leveraging Rancher Kubernetes Engine (RKE1) run Control Plane components as Docker containers and not as Pods managed by the Kubelet. 
+Therefore, the integration cannot autodiscover those containers and every endpoint needs to be manually specified in the `staticEndpoint` section of the integration configuration.
+
+Notice that the integration requires to be able to reach the different endpoints either connecting directly, with the available authentication methods (no-auth / token authentication / MTLS authentication), or thorugh a proxy.
+
+For example, in order to make reachable the `Scheduler` and the `Controller Manager` metrics you might need to add the [flag](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) `--authorization-always-allow-paths` allowing `/metrics` or `--authentication-kubeconfig` and `--authorization-kubeconfig` to enable token authentication.
```

Comment:
Are we using `Backticks` for control plane components elsewhere?

Reply:
mmm we do not